src/ui/FormMessageItemBody/FormInput.tsx

```diff
@@ -245,7 +245,7 @@ const FormInput = (props: InputProps) => {
                   ) : (
                     <>
                       <input
-                        type={layout}
+                        type={layout === 'number' ? 'text' : layout}
```

Comment:
Would you add `inputMode` instead?

Reply:
done!

Reply:
where?

Reply:
oops I added in widget. I just commited here too